protocol/x/subaccounts/keeper/subaccount_test.go

```diff
@@ -95,13 +95,21 @@ func assertSubaccountUpdateEventsInIndexerBlock(
 
 	// For each update, verify that the expected SubaccountUpdateEvent is emitted.
 	for _, update := range updates {
-		expecetedSubaccountUpdateEvent := indexerevents.NewSubaccountUpdateEvent(
+		expectedSubaccountUpdateEvent := indexerevents.NewSubaccountUpdateEvent(
 			&update.SubaccountId,
 			expectedUpdatedPerpetualPositions[update.SubaccountId],
 			expectedUpdatedAssetPositions[update.SubaccountId],
 			expectedSubaccoundIdToFundingPayments[update.SubaccountId],
 		)
-		require.Contains(t, subaccountUpdates, expecetedSubaccountUpdateEvent)
+
```

Comment:
`require.Equal(..)` explicitly logs out difference in the struct, which makes debugging easier.